README.md

```diff
@@ -223,8 +223,34 @@ For 1000, 5MB Files:
 
 *Of course, by looking at the pattern above, other number-size arrangements can be generated for further testing.*
 
+#### Variety of file types/sizes
 
-##### Large number of downloads
+*Prepare data variety*
+
+To test a variety of file types/sizes simply create a directory in `test-tree` such as `variety` and store a bunch of diffent files types in the range of a few kilo bytes to about 50 mega bytes. Ideally, a few images (`.png`, `.jpg`), document files (`.docx`, `.ppt`, `.xlxs`), video and sound files, and archive files.
+
+*Otherwise* 
+
+- You can download our test data stored in aws via `aws s3 cp s3://permanent-repos/test_files/critical-path.zip ./test-tree/variety`. (Access required via aws authentication)
+- It's a zip file, so remember to unzip: `unzip -d ./test-tree/variety ./test-tree/variety/critical-path.zip`
+- Delete the zip or move it to avoid it being part of the test data `rm ./test-tree/variety/critical-path.zip`
```

Comment:
I am thinking it's a good idea to just keep this data in this repository. It's less than 100MB and there's nothing private in it.

Additionally, 

- It would help me avoid the need to setup aws access [here](https://github.com/OpenTechStrategies/permanent-rclone-iac/blob/main/provision) just be able to download this data.
- Finally everything needed to run variety test would be inside this repo, so would also reduce setup here

cc : @jasonowen @slifty

Reply:
Adding this as an extra comment just in case we decide to remove it.